### PR TITLE
O3-1464: Create E2E automated workflow to test appointments feature in o3

### PR DESCRIPTION
**Pull request by pasindur99**
## Purpose
The pull request is to create appointments E2E test for O3. And this pull request is related to the issue ticket number [O3-1464](https://issues.openmrs.org/browse/O3-1464)

## Goals.
- Check the status of the requesting appointment End to End process.

## Approach
 - Create a feature file and a step definition file for the test.
 - Create GitHub workflow.
 - Migrate the test result to the dashboard.

## Note
*  Requesting appointments feature still under development.

### CODE REVIEW

**Comment by pasindur99** [comment]
This pull request is ready for review.
cc: @jayasanka-sack

File: packages/cypress/resources/features/refapp-3.x/14-appointments/appointments.feature [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+  @appointments
+    Scenario: The users should be able to make an appointment
+      When a user clicks on Appointments tab
+      Then the empty upcoming appointment table should displayed
+      When the user clicks on past appointments section
+      Then the empty past appointment table should displayed
+#  ToDo: Requests an appointment
+#      When a user requests an appointment
+#      Then the request should saved successfully
```
Comment (jayasanka-sack) [inline packages/cypress/resources/features/refapp-3.x/14-appointments/appointments.feature:15]: Is there any blockers on implementing this?

Comment (pasindur99) [reply packages/cypress/resources/features/refapp-3.x/14-appointments/appointments.feature:15]: In the appointment schedule forum, there are no data for selecting a service type. So we cannot request appointments for the patients. So this is a blocker for implementing the test further.
ss-
image

cc: @jayasanka-sack

Comment (jayasanka-sack) [reply packages/cypress/resources/features/refapp-3.x/14-appointments/appointments.feature:15]: Could you please raise this issue at the talk?

**Review by jayasanka-sack — APPROVED** [review]
Thanks @pasindur99 !

File: packages/cypress/resources/features/refapp-3.x/14-appointments/appointments.feature (hunk 1 above)
Comment (pasindur99) [reply packages/cypress/resources/features/refapp-3.x/14-appointments/appointments.feature:15]: Yeah, sure!

**Pull request merged**